go.mod

```diff
@@ -51,11 +51,11 @@ require (
 	github.com/raulk/go-watchdog v1.3.0
 	github.com/stretchr/testify v1.8.1
 	go.opencensus.io v0.24.0
-	go.uber.org/fx v1.18.2
-	go.uber.org/goleak v1.1.12
-	golang.org/x/crypto v0.3.0
+	go.uber.org/fx v1.19.0
```

Comment:
There’s something wrong with the dependencies here. There’s no reason to update fx and qtls in this PR.

Reply:
Should I revert the second commit, which was basically a `go get -u`?
Maybe you can update the other dependencies some other day :+1:

Reply:
Yes, please do. It’s good to keep PRs limited to one change at a time.

Reply:
I knew it wasn't clean when I did it, but I like to live dangerously :trollface: 
Should be fine now, I reverted the second commit :+1: